### ISSUE TITLE
Starting the electric sync service was confusing

### ISSUE THREAD

**Issue by lawik** [electric-sql/electric#731]
The documentation shows to use the `make start_dev_env` and `make shell` calls to get going.

Then you hit secure auth alg errors that require you to set some env vars. Those are clarified way further down in the documentation.

I assume you don't need to bother with this stuff if you use the Docker approach. But since I'm more comfortable with Elixir projects than Docker I figured I'd go the source route. This tripped me up for a minute.

**Comment by alco** [comment]
Thanks for bringing this up, @lawik! I'm preparing a few changes to be merged into `main` soon that will make the configuration error reporting much more descriptive. Specifically, once https://github.com/electric-sql/electric/pull/770 lands, it will be possible to start Electric from source with just `iex -S mix` out of the box.

As for the `make` commands mentioned in the README, it's high time for us to review and update those. We've not been using `make shell` ourselves for a long time now, and the only thing `make start_dev_env` does is start Postgres in a Docker container.

**Comment by msfstef** [comment]
@lawik since https://github.com/electric-sql/electric/pull/770 has been merged now and running Electric out of the box is simpler, is it ok to close this issue?

**Comment by lawik** [comment]
No idea :) If you think it is easier now, I assume it is.
